packages/ckeditor5-dev-stale-bot/bin/stale-bot.js

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const fs = require( 'fs-extra' );
+const chalk = require( 'chalk' );
+const createSpinner = require( './utils/createspinner' );
+const parseArguments = require( './utils/parsearguments' );
+const validateConfig = require( './utils/validateconfig' );
+const parseConfig = require( './utils/parseconfig' );
+const GitHubRepository = require( '../lib/githubrepository' );
+
+main().catch( error => {
+	console.error( '\n🔥 Unable to process stale issues and pull requests.\n', error );
+
+	process.exit( 1 );
+} );
+
+/**
+ * Launches the stale bot that searches the issues and pull requests to stale, unstale or close.
+ *
+ * @returns {Promise}
+ */
+async function main() {
+	const { configPath, dryRun } = parseArguments( process.argv.slice( 2 ) );
+
+	if ( !configPath || !await fs.exists( configPath ) ) {
+		throw new Error( 'Missing or invalid CLI argument: --config-path' );
+	}
+
+	const config = require( configPath );
+
+	validateConfig( config );
+
+	printWelcomeMessage( dryRun );
+
+	const githubRepository = new GitHubRepository( config.GITHUB_TOKEN );
+
+	const viewerLogin = await githubRepository.getViewerLogin();
+
+	const options = parseConfig( viewerLogin, config );
+
+	const spinner = createSpinner();
+	spinner.instance.start();
+
+	const searchResult = await search( githubRepository, options, spinner );
+	const {
+		issuesOrPullRequestsToStale,
+		issuesOrPullRequestsToClose,
+		issuesOrPullRequestsToUnstale
+	} = searchResult;
```

Comment:
```suggestion
	const {
		issuesOrPullRequestsToStale,
		issuesOrPullRequestsToClose,
		issuesOrPullRequestsToUnstale
	} = await search( githubRepository, options, spinner );
```

Reply:
This will not work as expected, because `searchResult` constant is also used at the end of `main()` function.